shared/fs/common/path-item-action/header-container.tsx

```diff
@@ -11,20 +10,13 @@ type OwnProps = {
 type StateProps = {
   _pathItems: Types.PathItems
 }
-type DispatchProps = {
-  loadFolderList: () => void
-  loadPathMetadata: () => void
-}
+type DispatchProps = {}
```

Comment:
we shouldn't type the return values of mapState or mapDispatch. TS should derive it fine